Toposes/gap/ToposDerivedMethods.gi

```diff
@@ -1030,14 +1030,37 @@ end :
   CategoryGetters := rec( range_cat := RangeCategoryOfHomomorphismStructure ),
   CategoryFilter := cat -> HasRangeCategoryOfHomomorphismStructure( cat ) );
 
+##
```

Comment:
@zickgraf: This derivation could be moved back to `CAP`.

Reply:
I'm fine with it staying here for now since I expect more subtleties, see https://github.com/homalg-project/CAP_project/pull/1470#discussion_r1344304865.

Reply:
Also fine for me.